strawberryfields/tdm/tdmprogram.py

```diff
@@ -113,7 +87,7 @@ def _get_mode_order(num_of_values, modes, N, timebins):
 
     """
     all_modes = []
-    for i in range(len(N)):
+    for i, _ in enumerate(N):
```

Comment:
Codefactor has started complaining (or maybe it always has) about using `for i in range(len(N)):` instead of `for i, _ in enumerate(N):`. I'm honestly happy either way.